src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -10,10 +10,13 @@ import { Platform, System_Array, Pointer, System_Object, System_String, HeapLock
 import { WebAssemblyBootResourceType } from '../WebAssemblyStartOptions';
 import { BootJsonData, ICUDataMode } from '../BootConfig';
 import { Blazor } from '../../GlobalExports';
+import { BINDINGType, CreateDotnetRuntimeType, DotnetModuleConfig, EmscriptenModule, MONOType } from 'dotnet';
 
-declare let Module: EmscriptenModule;
+// initially undefined and only fully initialized after createEmscriptenModuleInstance()
+export let BINDING: BINDINGType = <any>undefined;
+export let MONO: MONOType = <any>undefined;
```

Comment:
Shouldn't it be of type `MONOType | undefined` if it's sometimes undefined? Would that help other code to keep track of checking that it's become defined before using it?

Same with `BINDING` above.

Reply:
Actually I see how it would cause a lot of noise in the code below if you did that. The way you've done it looks good.

Reply:
Would you like to make asserts in each method ? If so, it's easy to do.

Reply:
I don't think we want to introduce any extra runtime checking in such low-level code (e.g., reading bytes from memory). So I'd vote for leaving this aspect as-is.